src/Components/Resource/ResourceDetails.tsx

```diff
@@ -25,7 +25,7 @@ const PageTitle = loadable(() => import("../Common/PageTitle"));
 
 export default function ResourceDetails(props: { id: string }) {
   const dispatch: any = useDispatch();
-  let initialData: any = {};
+  const initialData: any = {};
```

Comment:
Avoid `any` type

Reply:
Removed this change as out of score for this PR. Since specifying type here would need other changes. Would resolve this issue separately.